R/record_versions.R

```diff
@@ -41,15 +37,25 @@ record_versions <- function(
 #' @title Get the current versions of packages
 #' @export
 #' @keywords internal
-#' @description Get the current versions of packages in the repos.
+#' @description Get the current versions of packages in the repo.
 #' @return A data frame of packages with their current versions and hashes.
 #' @inheritParams record_versions
 get_current_versions <- function(
-  repos = "https://r-releases.r-universe.dev"
+  repo = "https://r-releases.r-universe.dev"
 ) {
-  out <- utils::available.packages(repos = repos)
-  out <- out[, c("Package", "Version", "MD5sum")]
-  out <- as.data.frame(out)
+  listing <- paste(
+    contrib.url(repos = repo, type = "source"),
+    "PACKAGES.json?fields=RemoteSha",
+    sep = "/"
+  )
```

Comment:
can use your neat `file.path()` trick here

Reply:
done in 8cd45a33ce3624496fce57ff479bcdd60fa28116